src/rc-local-generator/rc-local-generator.c

```diff
@@ -53,13 +53,19 @@ static int run(int argc, char *argv[]) {
         if (argc > 1)
                 arg_dest = argv[1];
 
-        if (access(RC_LOCAL_SCRIPT_PATH_START, X_OK) >= 0) {
+        if (access(RC_LOCAL_SCRIPT_PATH_START, X_OK) < 0)
+                log_full_errno(errno == ENOENT ? LOG_DEBUG : LOG_WARNING, errno,
+                               RC_LOCAL_SCRIPT_PATH_START " is not executable: %m");
```

Comment:
maybe say "… does not exist or is not executable"?

Reply:
I don't think it matters enough. "... is not executable: no such file or directory" is clear enough, and it only appears with debug logging.

Reply:
true